external-import/comlaude/src/main.py

```diff
@@ -87,6 +88,12 @@ def _generate_dynamic_custom_properties(helper, domain_object, score):
     return domain_name, custom_properties
 
 
+AUTHOR_NAME = "Comlaude"
```

Comment:
In the config.yml we can re-use the connector name, by adding a get_config_variable:
self.connector_name = get_config_variable(
 "CONNECTOR_NAME", ["connector", "name"], self.config,
)